lib/features/give/pages/bt_scan_page.dart

```diff
@@ -22,13 +22,26 @@ class BTScanPage extends StatefulWidget {
 class _BTScanPageState extends State<BTScanPage> {
   bool _isVisible = false;
 
+  // We need to keep track of the scanning state to prevent scans when the user is going into the manual flow
+  bool _isSearching = false;
```

Comment:
The whole class is already private, is it needed to have these ones private as well? 🤔